### PR TITLE
Qor integration test

### PR DESCRIPTION
**Pull request by raven-chen**

### CODE REVIEW

File: admin/context_test.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package admin_test
```
Comment (shenzhaoyan) [inline admin/context_test.go]: The package name should be `admin`?

Comment (raven-chen) [inline admin/context_test.go]: we haven't decided where to put unit tests. an independent  package or follow go convention.

Comment (bodhi) [inline admin/context_test.go]: Why *not* follow Go conventions?

Comment (raven-chen) [inline admin/context_test.go]: because it takes time to switch package for these old tests. I'm working on CI now. will migrate them later.

Comment (bodhi) [inline admin/context_test.go]: > I'm working on CI now.

Document on Qortex please!

Comment (raven-chen) [inline admin/context_test.go]: er ?

Comment (bodhi) [inline admin/context_test.go]: * What CI service have you looked at (I saw you signed up for a couple)? 
* Did you pick one?
* How about `cbr.theplant-dev.com` or `ci.theplant-dev.com`?
* Do you have any other goals for "working on CI"?

Comment (raven-chen) [inline admin/context_test.go]: created [here](https://qortex.com/theplant/#groups/52156d1a3c58162268002cea/entry/555310fd8d93e322ff01ab5b)

**Pull request merged**